src/Middleware/HttpLogging/src/HttpLoggingServicesExtensions.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.HttpLogging;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    /// <summary>
+    /// Extension methods for the HttpLogging middleware.
+    /// </summary>
+    public static class HttpLoggingServicesExtensions
+    {
+        /// <summary>
+        /// Adds HTTP Logging services.
+        /// </summary>
+        /// <param name="services">The <see cref="IServiceCollection"/> for adding services.</param>
+        /// <param name="configureOptions">A delegate to configure the <see cref="HttpLoggingOptions"/>.</param>
+        /// <returns></returns>
+        public static IServiceCollection AddHttpLogging(this IServiceCollection services, Action<HttpLoggingOptions> configureOptions)
```

Comment:
Do we need an overload without `configureOptions`? I would be ok making it optional.

Reply:
Calling AddHttpLogging is optional, it only sets options. We don't need another overload or an optional arg for now.

Reply:
How do you feel about calling this `ConfigureHttpLogging` in that case? Mvc does it in a few places for APIs meant for configuring but not adding a feature.

Reply:
I think I do prefer that name, @davidfowl @Tratcher to double check.

Reply:
+1 I prefer that name.

Reply:
We use Add most places. Where do we use Configure?

Reply:
- https://github.com/dotnet/aspnetcore/blob/61cdc605a8a5ad8af7189ebb21b4a996bc77833d/src/Servers/Kestrel/Kestrel/src/WebHostBuilderKestrelExtensions.cs#L104
- https://github.com/dotnet/aspnetcore/blob/61cdc605a8a5ad8af7189ebb21b4a996bc77833d/src/Mvc/Mvc.Core/src/DependencyInjection/MvcCoreMvcBuilderExtensions.cs#L131
- https://github.com/dotnet/aspnetcore/blob/61cdc605a8a5ad8af7189ebb21b4a996bc77833d/src/Mvc/Mvc.Core/src/DependencyInjection/MvcCoreMvcBuilderExtensions.cs#L201
- https://github.com/dotnet/aspnetcore/blob/61cdc605a8a5ad8af7189ebb21b4a996bc77833d/src/Identity/Core/src/IdentityServiceCollectionExtensions.cs#L110


But here's a counter example:

https://github.com/dotnet/aspnetcore/blob/61cdc605a8a5ad8af7189ebb21b4a996bc77833d/src/Middleware/WebSockets/src/WebSocketsDependencyInjectionExtensions.cs

Reply:
Fair

Reply:
Alright, I'll revert to Add for now.